include/IECoreMaya/ToMayaPlugConverter.h

```diff
@@ -64,11 +65,13 @@ class IECOREMAYA_API ToMayaPlugConverter : public ToMayaConverter
 
 		/// \todo Implement this as a genuine factory which creates subclasses.
 		/// \todo Should be ConstPtr not const Ptr.
+		template <typename DataType, typename UnitType>	MStatus convertAttr(MPlug &plug) const;
 		static ToMayaPlugConverterPtr create( const IECore::ObjectPtr src );
 
 	private :
 
 		ToMayaPlugConverter( IECore::ConstObjectPtr obj );
+		template <typename DataType, typename UnitType> static MStatus setPlugValue(MPlug &plug, boost::intrusive_ptr<const IECore::TypedData<DataType> > data);
```

Comment:
Is `boost::intrusive_ptr<const IECore::TypedData<DataType> >` different to `IECore::TypedData<DataType>::ConstPtr`? Also, do we even need a smart pointer here or is `setPlugValue` just going to grab the value momentarily without taking ownership over the TypedData?